HHS/OPRE-OPS
# OPS-1525 workflow notification fixes and improvements

## What changed

* adds notes from the rejection to Notification message (with Markdown formatting)
* adds notification of rejection to the Project Officer
* add Markdown formatting for rendering LogItems
* some refactoring for sending notifications requesting approval to all directors to the BLIs in the package in workflow-submit (using hard-coded IDs since there's none in division.division_director_id

## Issue

#1525 

## How to test

For rejections notifications, create workflows, reject and view notifications to see the notes or view the Notifications table.
For approval request notifications, Admin Demo is now one of hard coded directors.

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/127965976/13750293-620c-43de-bc4e-9bf65a6df539)


## Definition of Done Checklist


## Links

_If relevant, e.g. for a link to a piece of markdown documentation_
